pkg/chains/formats/slsa/v2alpha2/internal/pipelinerun/pipelinerun.go

```diff
@@ -94,6 +94,15 @@ func internalParameters(pro *objects.PipelineRunObject) map[string]any {
 // externalParameters adds the pipeline run spec
 func externalParameters(pro *objects.PipelineRunObject) map[string]any {
 	externalParams := make(map[string]any)
+	// add top level pipeline config
+	if p := pro.Status.Provenance; p != nil && p.RefSource != nil && pro.Spec.PipelineRef != nil {
```

Comment:
It seems to me that `pro.Spec.PipelineRef != nil` condition is not needed here. Only relying on the Provenance.RefSource field to report the origin should be sufficient here.

Reply:
Actually, we need to check if the resolver is not the "ClusterResolver" as well. The reason is that we plan to provide a digest (`checksum`) of in-cluster pipeline and task too. Here we only want to capture `remote` reference right?

Reply:
I'm thinking it should be this:
```
if p := pro.Status.Provenance; p != nil && p.RefSource != nil &&  pro.Spec.PipelineRef != nil && pro.Spec.PipelineRef.Resolver != nil && pro.Spec.PipelineRef.ClusterResolver == nil
```
WDYT?